allure-generator/webpack.config.js

```diff
@@ -7,8 +7,6 @@ const postcssLoader = {
     loader: 'postcss-loader',
     options: {
         plugins: [
-            require('postcss-import'),
-            require('precss')({'import': {disable: true}}),
             require('autoprefixer')
```

Comment:
PostCSS is still used, as we want to have autoprefixer. (To have less pain supporting IE and Safari)